src/graphics/render_window.rs

```diff
@@ -342,6 +492,23 @@ impl RenderWindow {
     /// # Arguments
     /// * position - New position of the window, in pixels
     ///
+    /// # Usage Example
+    ///
+    /// ```no_run
+    /// # use sfml::window::{Event, Style};
+    /// # use sfml::graphics::RenderWindow;
+    /// # use sfml::system::Vector2;
+    /// # // Create a new window with SFML window as name
+    /// # let mut window = RenderWindow::new((800, 600),
+    /// #                              "SFML window",
+    /// #                              Style::CLOSE,
+    /// #                              &Default::default());
+    /// window.set_position(Vector2::new(100, 400));
+    /// use std::{thread, time::Duration};
+    /// // You need to wait for the OS the set the window's position before checking
+    /// thread::sleep(Duration::from_secs(1));
```

Comment:
I think you could lower the sleep to 250 milliseconds. It's reasonable, and doesn't increase test times as much as 1 second.